bundles/binding/org.openhab.binding.temperlan/src/main/java/org/openhab/binding/temperlan/internal/hardware/Plug.java

```diff
@@ -0,0 +1,68 @@
+package org.openhab.binding.temperlan.internal.hardware;
+
+import java.io.BufferedInputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.net.InetSocketAddress;
+import java.net.Socket;
+
+public class Plug
+{
+  public static final int READTIMEOUT = 2000;
+  public static final int TIMEOUT = 5000;
+  byte[] data = new byte[1000];

```

Comment:
Is there a more formal origin for the decision to use 1000 for this byte array size?